<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QDialog" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tw_workload">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_year">
      <attribute name="title">
       <string>Год</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="hf_year">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_month">
      <attribute name="title">
       <string>Месяц</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_month"/>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="hf_month">
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lb_airport">
     <property name="text">
      <string>Аэропорт:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
